polytracker/src/polytracker/polytracker.cpp

```diff
@@ -13,43 +13,16 @@ EARLY_CONSTRUCT_EXTERN_GETTER(taintdag::PolyTracker, polytracker_tdag);
 
 static std::atomic_flag polytracker_init_flag = ATOMIC_FLAG_INIT;
 
-static bool polytracker_is_initialized() {
-  return polytracker_init_flag.test(std::memory_order_relaxed);
-}
-
 static void polytracker_initialize() {
   polytracker_init_flag.test_and_set(std::memory_order_relaxed);
 }
 
-extern "C" taintdag::Functions::index_t
-__polytracker_log_func_entry(char *name, uint16_t len) {
-  if (!polytracker_is_initialized()) {
-    return 0;
-  }
-  return get_polytracker_tdag().function_entry({name, len});
-}
-
-extern "C" void
-__polytracker_log_func_exit(taintdag::Functions::index_t func_index) {
-  if (!polytracker_is_initialized()) {
-    return;
-  }
-  get_polytracker_tdag().function_exit(func_index);
-}
-
 extern "C" dfsan_label __polytracker_union_table(const dfsan_label &l1,
                                                  const dfsan_label &l2) {
-  if (!polytracker_is_initialized()) {
```

Comment:
these were removed because they were added to fix the stdin test and did not do anything about it. I can put them back if we want them, but they don't _seem_ necessary.